src/main/java/hudson/scm/subversion/NoopUpdater.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2004-2010, Sun Microsystems, Inc., Kohsuke Kawaguchi, Fulvio Cavarretta,
+ * Jean-Baptiste Quenot, Luca Domenico Milanesio, Renaud Bruyeron, Stephen Connolly,
+ * Tom Huybrechts, Yahoo! Inc., Manufacture Francaise des Pneumatiques Michelin,
+ * Romain Seguy
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package hudson.scm.subversion;
+
+import hudson.Extension;
+import hudson.scm.SubversionSCM.External;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * {@link WorkspaceUpdater} that doesn't touch working copy.
+ *
+ * @author Moisei Rabinovich
+ */
+public class NoopUpdater extends WorkspaceUpdater {
+    private static final long serialVersionUID = 1951258464864424355L;
+
+    @DataBoundConstructor
+    public NoopUpdater() {
+    }
+
+    @Override
+    public UpdateTask createTask() {
+        return new TaskImpl();
+    }
+
+    public static class TaskImpl extends UpdateTask {
+        private static final long serialVersionUID = -5966470969352844330L;
+
+        @Override
+        public List<External> perform() throws IOException, InterruptedException {
+            return new ArrayList<External>();
+        }
+
+
+    }
+
+    @Extension(ordinal = 100) // this is the default, so given a higher ordinal
```

Comment:
And since `N` precedes `U`, it was the first choice in the list, which of course breaks every UI-based test!